src/Translation_unit.ml

```diff
@@ -272,6 +272,9 @@ let parse_print (XUnit xunit) (conf : Conf.t) ~input_name ~input_file ic
         exe input_file ;
       Format.eprintf
         "  BUG: formatting did not stabilize after %i iterations.\n%!" i
+  | Ocamlformat_bug (Sys_error msg)
```

Comment:
It feels wrong to use the `Ocamlformat_bug` exception if this not considered to be an ocamlformat bug.